### DISCUSSION TITLE
Differentiate between "locked" and "non-interactive" ?

### DISCUSSION THREAD

**Discussion by aadrian** [bcakmakoglu/vue-flow#1617] [General]
Would it possible to differentiate between "locked"  and "non-interactive" ?

Currently when "interactive" is toggled down,  `onClick` events on nodes and edges are not dispatched anymore :( .

It is very useful that when the "lock" icon is toggled, the layout of the diagram can't be changed or edited (so read-only, or 'view/display' mode), however the click events are still needed in most cases ! 

Thank you.

**Comment by bcakmakoglu** [comment]
Mh... actually this might be considered a bug because the pointer events should still be applied *if* a handler like `@node-click` is passed.

If you're doing it from inside a custom node you would have to apply `pointer-events: all` yourself though.

**Comment by bcakmakoglu** [comment]
This is fixed in `1.41.2`.

**Reply by aadrian** [reply]
> This is fixed in `1.41.2`.

Fantastic ! 

Thank you very much !
